docs/annihilation.rst

```diff
@@ -58,8 +58,82 @@ In order to properly assign parameters to the particles in the hybrid system, th
 - Unique new: These atoms are not in the map, and are present only in the new system
     - Example: The chlorine atom that appears when a benzene is transformed into a chlorobenzene.
     
-Creation of Force Terms
-^^^^^^^^^^^^^^^^^^^^^^^
+Treatment of different interaction groups
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 For each supported force, we have to create at least one custom force, as well as an unmodified force. In general, the interactions are handled as follows:
 
+- Environment-Environment: These interactions are handled by regular unmodified forces, as their parameters never change.
+
+- Environment-{Core, Unique old, Unique new}: These interactions are handled by Custom*Forces. For interactions involving the core atoms, the interaction scales from the parameter in the old system to the parameter in the new system as lambda is switched from 0 to 1. For interactions involving the unique atoms, the interaction scales from full to zero as lambda switches from 0 to 1 in unique old atoms, and vice versa for unique new atoms.
+
+- Core-Core interactions: These interactions scale from old system parameters to new system parameters as lambda is switched from 0 to 1.
+
+- Core-{unique old, unique new} interactions: These interactions scale from the parameters in the old system to the parameters in the new system as lambda is switched from 0 to 1. This means that unique atoms always maintain intramolecular interactions.
+
+- Unique-old and unique-new interactions: These are disabled.
+
+- intra-Unique atoms: These interactions are always enabled, and as such are handled by an unmodified force.
+
+
+Force terms
+^^^^^^^^^^^
+
+Given the above division of atoms into different groups, the assignment of interactions to different force classes is straightforward. 
+
+
+Bonds
+"""""
+
+Bonds are handled with a HarmonicBondForce and a CustomBondForce.
```

Comment:
You might want to back-tick protect OpenMM classes like `HarmonicBondForce`

I think sphinx uses two backticks before and after: http://www.sphinx-doc.org/en/stable/rest.html